src/irmin-pack/unix/gc.mli

```diff
@@ -54,7 +54,7 @@ module Make (Args : Gc_args.S) : sig
 
   val cancel : t -> bool
 
-  val finalise_without_swap : t -> (int63 * int63) Lwt.t
+  val finalise_without_swap : t -> (int63 * int63 * int63) Lwt.t
```

Comment:
I went with producing a `gced` control file status here directly since it has all the required values for the store to then create the snapshot... but let me know if you think it's worse!

Reply:
I kinda like it!

My only point of consideration is thinking about where the "knowledge" of constructing the Gced status ought to live.

If we wanted to embrace the Gc module having the knowledge of how to construct a control file Gced status after Gc finishes, we could also change the file manager's `swap` function to accept the status instead of creating it itself. But that is definitely for a separate PR!